### ISSUE TITLE
Add init/scaffold stage

### ISSUE THREAD

**Issue by gumanista** [skilld-labs/skilld-docker-container#50]
Right now we are doing 'silent' project initiation by creating missed files, copying files, etc
It would make sense to have it as a separate task, not running all the times we are running a project. On this stage we can:
- set project variables: theme, name, etc
- set COMPOSE_FILE variable to include necessary docker-compose files (for example os-specific)
- traefik usage
etc

**Comment by andypost** [comment]
Good idea to separate init stage butbwe still need some validation before each execution

**Comment by gumanista** [comment]
Agree, but don't need to take decisions 'silently'. Just provide recommendations. Probably this should be an additional issue.
